app/default_overrides.go

```diff
@@ -249,16 +249,14 @@ func DefaultBlockParams() *tmproto.BlockParams {
 	}
 }
 
-// DefaultEvidenceParams returns a default EvidenceParams with a MaxAge
-// determined using a goal block time.
+// DefaultEvidenceParams returns the evidence params defined in CIP-37. The
+// evidence parameters are not modifiable by governance so a consensus breaking
+// release is needed to modify them.
```

Comment:
```suggestion
// DefaultEvidenceParams returns the evidence params defined in CIP-37. The
// evidence parameters are not modifiable by governance so a consensus breaking
// release is needed to modify them or parameters returned from this function.
```

Reply:
https://github.com/celestiaorg/celestia-app/pull/5332/commits/0c077a88d8832efd0be4be4d3a4816791c1dae73

Reply:
the commit posted solves the issue imo thanks!

the reasoning for adding that last bit originally was just to be extra explicit. perhaps a warning would have been better to simply indicate that we use this function in the state machine.

> The evidence parameters are not modifiable by governance so a consensus breaking release is needed to modify them

to me, this could be interpreted simply as a statement that a hardfork is needed to change the consensus commands, not that if you change something in this function it will introduce a consensus breaking change.